.github/workflows/overcloud-host-image-build.yml

```diff
@@ -243,6 +342,16 @@ jobs:
           KAYOBE_VAULT_PASSWORD: ${{ secrets.KAYOBE_VAULT_PASSWORD }}
         if: inputs.rocky9
 
+      - name: Show last error logs
```

Comment:
Is this just for testing?

Reply:
I initially added it for testing before I could copy back the full logs, but I was going to leave it in since it's very convenient to just check for obvious errors.